language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaTextDocumentService.java

```diff
@@ -209,7 +211,28 @@ public BallerinaTextDocumentService(BallerinaLanguageServer ballerinaLanguageSer
 
     @Override
     public CompletableFuture<List<? extends Location>> references(ReferenceParams params) {
-        return CompletableFuture.supplyAsync(() -> null);
+        return CompletableFuture.supplyAsync(() -> {
+            ReferenceParams ref = params;
```

Comment:
Shall we remove the redundant local variable?

Reply:
will do ... thank you